website/addons/wiki/model.py

```diff
@@ -34,6 +35,47 @@ class AddonWikiNodeSettings(AddonNodeSettingsBase):
 
     complete = True
     has_auth = True
+    is_publicly_editable = fields.BooleanField(default=False, index=True)
+
+    def set_editing(self, permissions, auth=None, node=None, save=False, log=False):
+        """Set the editing permissions for this node.
+
+        :param permissions: A string, either 'public' or 'private'
+        :param auth: All the auth informtion including user, API key
+        :param node: Node to which self belongs,
```

Comment:
Why is this necessary at all?

It seems like you would always be able to access it at `self.owner`.